var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -96,6 +96,8 @@ class Trilinos(CMakePackage, CudaPackage):
             description='Compile with HDF5')
     variant('hypre',        default=False,
             description='Compile with Hypre preconditioner')
+    variant('libx11',         default=False,
```

Comment:
Sorry about the removal. Can you change the variant name to its previous incarnation, `x11`, which is more frequently used in the other spack packages as well?

Reply:
updated I had previously misunderstood your TPL section or you updated it to allow variants to have different names than dependencies